all-repos.yaml

```diff
@@ -221,3 +221,4 @@
 - https://github.com/sirwart/ripsecrets
 - https://github.com/bagerard/graphviz-dot-hooks
 - https://github.com/omnilib/ufmt
+- https://github.com/slobberchops/conhead
```

Comment:
it probably makes sense to collaborate with existing and more battle-tested implementations of this such as https://github.com/Lucas-C/pre-commit-hooks#insert-license